src/benchmarks/micro/corefx/System.Runtime/Perf.Int32.cs

```diff
@@ -26,11 +26,15 @@ public class Perf_Int32
         [Benchmark]
         [ArgumentsSource(nameof(Int32Values))]
         public string ToString(int value) => value.ToString();
-        
+
+        [Benchmark]
+        [ArgumentsSource(nameof(StringValues))]
+        public int Parse(string value) => int.Parse(value);
+
 #if !NETFRAMEWORK && !NETCOREAPP2_0 // API added in .NET Core 2.1
         [Benchmark]
         [ArgumentsSource(nameof(StringValues))]
-        public int Parse(string value) => int.Parse(value.AsSpan());
+        public int ParseSpan(string value) => int.Parse(value.AsSpan());
```

Comment:
this is a breaking change (benchmark rename) but totally needed  and reasonable 👍